discover/index.html

```diff
@@ -23,80 +23,64 @@
 </div>
 <h2 class="text-left"><i class="fas fa-fw fa-plane"></i> Getting Started</h2>
 <p class="pop text-center">
-  <strong>Anyone</strong> wanting to use the AGRC imagery and base map services from Discover needs to create a <strong><em>free</em></strong> account and receive a <em>quad-word</em> connection link.
+  <strong>Anyone</strong> can use the AGRC imagery and base map services from Discover! To use these services, all you need is a <strong><em>free</em></strong> account and a <em>quad-word</em> connection link.
```

Comment:
(Here I was thinking a more active voice/direct address might be more effective.)